### ISSUE TITLE
[Syncing] form/FormTimeField

### ISSUE THREAD

**Issue by yedhink** [bigbinary/neeto-ui#84]
https://github.com/bigbinary/neeto-help-web/blob/master/app/javascript/Admin/Components/form/FormTimeField.jsx

Sync required.

@yedhink _a

**Comment by edwinbbu** [comment]
@yedhink I have created a component TimeInput in PR https://github.com/bigbinary/neeto-help-web/pull/8766. Once the PR is merged you can sync the component. 
Hence keeping the issue on hold until PR is merged.

**Comment by goutham-subramanyam** [comment]
Closing this in favour of https://github.com/bigbinary/neeto-ui/issues/113
